python_scripts/make_merged_CDE_table.py

```diff
@@ -0,0 +1,50 @@
+#!/bin/env python
```

Comment:
@gregpoore are you planning to run QIIME metadata file generation on CGC or locally? I think it should be OK for us to use Python 3.